.travis.yml

```diff
@@ -19,13 +19,18 @@ matrix:
         - php: hhvm
 
 before_install:
+    - if [ -f "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" ]; then mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini-disabled"; fi;
     - composer self-update
+    - if [ -n "$GITHUB_OAUTH_TOKEN" ]; then composer config --global github-oauth.github.com ${GITHUB_OAUTH_TOKEN}; fi;
```

Comment:
If it's not configured then this command will do nothing. Composer still falls back to doing git clone...